src/test/java/com/google/api/gax/testing/LocalServiceHelper.java

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.api.gax.testing;
```

Comment:
I think this should actually go under src/main/java instead of src/test/java, since this isn't a test, and someone might want to use it for something besides unit testing. Also, you would want to include the other general classes necessary for testing, e.g. GCloudEmulatorRunner, DownloadableEmulatorRunner, and others.